ui/app/AppLayouts/Profile/views/wallet/AccountView.qml

```diff
@@ -29,8 +32,11 @@ ColumnLayout {
 
     QtObject {
         id: d
-        property bool watchOnlyAccount: keyPair && keyPair.pairType === Constants.keycard.keyPairType.watchOnly
-        property bool privateKeyAccount: keyPair && keyPair.pairType === Constants.keycard.keyPairType.privateKeyImport
+        readonly property bool watchOnlyAccount: keyPair && keyPair.pairType ? keyPair.pairType === Constants.keycard.keyPairType.watchOnly: ""
```

Comment:
@Khushboo-dev-cpp shouldn't be  any bool value returned for the else block instead of `""`? The same for the line below.

Reply:
@Khushboo-dev-cpp also keypair type cannot be ever a watch only type. Only an account type can be watch only.

Reply:
Ive added keyPairs  for Profile  and in that I have clubbed all watch only walletAccounts into one keypair, for the sake of UI and hence this works fine

Reply:
I see the point, but definitely looks confusing when somebody see keypair with watch only accounts. :)
We should named that model somehow different then, like "groupedAccounts" or so.